crates/orchestrator/src/config.rs

```diff
@@ -49,6 +49,31 @@ pub struct Config {
     alerts: Box<dyn Alerts>,
 }
 
+/// `ProviderConfig` is an enum used to represent the global config built
+/// using the settings provider. More providers can be added eg : GCP, AZURE etc.
+pub enum ProviderConfig {
+    AWS(Box<SdkConfig>),
+    INVALID,
```

Comment:
why do we need this?

Reply:
abhi code me jha pr me match kr rha hu providerconfig ko vha pr I need to handle the other cases with a panic. so when it was alone AWS it was giving code unreachable error. So that's why added this enum temporarily.

Reply:
that's not the correct design. you don't need add another `enum` so that you can panic. if there was only AWS over here, the user couldn't pass anything else, the code won't compile so there's no issue